numba/tests/test_np_functions.py

```diff
@@ -709,7 +781,19 @@ def _check(k):
         for k in 1.5, True, np.inf, [1, 2]:
             _check(k)
 
-    def _triangular_matrix_tests(self, pyfunc):
+    def _triangular_matrix_tests_m(self, pyfunc):
+        cfunc = jit(nopython=True)(pyfunc)
+
+        def _check(arr):
+            expected = pyfunc(arr)
+            got = cfunc(arr)
+            # TODO: Contiguity of result not consistent with numpy
```

Comment:
Is this a new problem or just a note about an existing one?

Reply:
The latter - just a note about existing behaviour (on my list to revisit in a quiet moment).

Reply:
Great, thanks for confirming.